### ISSUE TITLE
Migration files missing for some models

### ISSUE THREAD

**Issue by jjliang** [jjliang/databaseui#38]
Since the migration files don't exists db:migrate does not create the tables properly so we should have that unless we want to run rake db:schema:load when building the project

**Comment by diceydawg** [comment]
John ran a few schema dumps so we don't need many of the migrations anymore.  We'll need to clean out the irrevelant migrations

**Comment by jjliang** [comment]
hum... I can disagree with Dice, but anyways I redid the migration files added the few that are missing. I did test the current database already, so if you drop the database in mysql, it would be rake db:create, rake db:migrate, rake db:seed
and it works I had to change the node to non_itis relationship as the has_one relationship broke rails admin, so I just kept the relationship that non_itis belongs to node. 
rake db:create does not run the schema.rb file so leaving the migration files to create tables in the folder. However, I did clear out some migrations and put them in the creation of the tables
